src/ndarray/ndarray.cc

```diff
@@ -639,25 +643,35 @@ void NDArray::Save(dmlc::Stream *strm) const {
 }
 
 bool NDArray::Load(dmlc::Stream *strm) {
```

Comment:
then it requires to change here: https://github.com/ZihengJiang/mxnet/blob/tvm/include/mxnet/ndarray.h#L337
ok?